PluginsAndFeatures/azure-toolkit-for-intellij/.gitignore

```diff
@@ -49,4 +49,5 @@ gen-external-apklibs/
 /azure-intellij-plugin-azure-sdk/.idea/
 
 # IntelliJ
-.idea/
\ No newline at end of file
+.idea/
+/.idea/
```

Comment:
remove

Reply:
Also add one empty line at the end of file for removing the red icon

Reply:
there was a .idea directory on the outer level that wasn't being ignored (it got pushed in another PR) -- but I deleted it so I'll take out this second /.idea directory and hope for the best